iotdb-1.1/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb110/IoTDBSessionBase.java

```diff
@@ -211,7 +212,13 @@ protected Status executeQueryAndGetStatus(String sql, Operation operation) {
                       }
                       records.add(record);
                     }
+                  } else {
+                    SessionDataSet.DataIterator iterator = sessionDataSet.iterator();
+                    while (iterator.next()) {
```

Comment:
Is the iterator only used to count this sessionDataSet's size ?

Reply:
used to simulate traversal operations of real users

Reply:
would real users typically construct RecordRows ?